rskj-core/src/test/java/co/rsk/peg/RegisterBtcTransactionIT.java

```diff
@@ -163,6 +172,55 @@ void registerBtcTransaction_whenLegacyBtcTransactionWithNegativeHeight_shouldNot
         assertEquals(expectedReceiverBalance, repository.getBalance(rskReceiver));
     }
 
+    @Test
+    void registerBtcTransaction_whenLegacyBtcTransactionFromAMultiSig_shouldRefundTheFunds() throws Exception {
+        // Arrange
+        BtcTransaction bitcoinTransaction = createMultiSigPegInTransaction(federationSupport.getActiveFederation().getAddress(), minimumPeginValue);
```

Comment:
nit: Since we are going to use `bitcoinTransaction` a lot, maybe adopt a shorter name such as `btcTx` or `btcTransaction`.

Reply:
done!